src/env/prod/terraform.tfvars

```diff
@@ -0,0 +1,171 @@
+apim_notification_sender_email = "info@pagopa.it"
+apim_publisher_email           = "io-operations@pagopa.it"
+apim_publisher_name            = "PagoPa Centro Stella DEV"
+apim_sku                       = "Developer_1"
+app_gateway_host_name          = "api.cstar-dev.pagopa.it"
+azdoa_scaleset_li_public_key   = "sensitive"
+balanced_proxy_ip              = "127.0.0.1"
+# https://www.davidc.net/sites/default/subnets/subnets.html?network=10.230.8.0&mask=21&division=31.d6627231
+cidr_vnet              = ["10.230.8.0/21"]
+cidr_subnet_k8s        = ["10.230.8.0/22"]
+cidr_subnet_appgateway = ["10.230.12.0/24"]
+cidr_subnet_apim       = ["10.230.13.0/26"]
+cidr_subnet_db         = ["10.230.13.64/26"]
+cidr_subnet_eventhub   = ["10.230.13.128/27"]
+cidr_subnet_azdoa      = ["10.230.13.160/27"]
+cidr_subnet_jumpbox    = ["10.230.13.192/27"]
+
+db_sku_name       = "GP_Gen5_2"
+db_enable_replica = true
+dns_zone_prefix   = "cstar"
+ehns_sku_name     = "Standard"
+enable_azdoa      = true
+env_short         = "p"
+eventhubs = [
+  {
+    name              = "bpd-citizen-trx"
+    partitions        = 1
```

Comment:
Use correct values for PROD

Reply:
I've just created the DNS public zone: apply with target. I've not created all resources yet.